modules/backend/classes/AuthManager.php

```diff
@@ -96,6 +96,29 @@ public function registerPermissions($owner, array $definitions)
         }
     }
 
+    /**
+     * Removes a single back-end permission
+     * @param string $owner Specifies the menu item's owner plugin or module in the format Vendor/Module.
```

Comment:
Can you fix this docblock (and the one for registerPermissions) to say `Specifies the permissions' owner plugin or module in the format Author.Plugin`